.github/workflows/test_annotate_params.yml

```diff
@@ -0,0 +1,28 @@
+name: test annotate_params_unittest.py
+
+on:
+  push:
+    paths:
+      - 'Tools/scripts/annotate_params.py'
+      - 'Tools/scripts/annotate_params_unittest.py'
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    container:
+      image: ardupilot/ardupilot-dev-clang:v0.1.3
+      options: --user 1001
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up Python
```

Comment:
we don't need the python setup nor Install dependencies as this is already done by the base image

Reply:
Done, I think